drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/models/AbstractScesimGridModel.java

```diff
@@ -603,6 +603,17 @@ public void selectRow(int rowIndex) {
         IntStream.range(0, columns).forEach(columnIndex -> selectCell(rowIndex, columnIndex));
     }
 
+    /**
+     * It selects the current selected header cells group
+     */
+    public void reselectCurrentHeaderCell() {
```

Comment:
To be honest I don't think the fix should be done here. The header of the columns is merged so I expect to have the selection updated out of the box by grid code.
If this is not working the fix should be done directly on [uberfire-wires-core-grids](https://github.com/kiegroup/appformer/tree/master/uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids).
Wdyt?

@manstis 
Do you have any advice?

Reply:
@danielezonca Ideally, it would be the best. The problem I see is that the current behavior of selecting spanned cells, is already defined in scesim module. If we choice to move the logic on uberfire-wires-core-grids, fine for me but it should be done moving all of the behavior, not only this PR.